sites/website/docusaurus.config.js

```diff
@@ -15,11 +15,13 @@ module.exports = {
     ],
     themes: [require.resolve("@docusaurus/theme-live-codeblock")],
     themeConfig: {
+        defaultDarkMode: true,
         navbar: {
             logo: {
                 alt:
                     "Line drawing of a small moon orbiting around a planet with the words FAST next to it",
-                src: "https://static.fast.design/assets/fast-inline-logo.png",
+                srcDark: "https://static.fast.design/assets/fast-inline-logo.png",
+                src: "https://static.fast.design/assets/fast-inline-logo-light.png",
```

Comment:
The name you have above would be the wrong path, you need to rename the png file to `fast-inline-logo-light.png`

Reply:
This should be right... the existing image is for dark mode. That image is now mapped to `srcDark`. The new file is for light mode (`src` instead of `srcDark`)